lib/screens/main_screen/main_screen.dart

```diff
@@ -53,12 +61,24 @@ class MainScreen extends StatelessWidget {
               medicationsRepository: context.read<MedicationsRepository>(),
             )..add(const MedicationsListSubscriptionRequested()),
           ),
+          BlocProvider<TherapiesListBloc>(
+            create: (BuildContext context) => TherapiesListBloc(
+              therapiesRepository: context.read<TherapiesRepository>(),
+            )..add(const TherapiesListSubscriptionRequested()),
+          ),
         ],
         child: MaterialApp(
           title: 'PICOS',
           localizationsDelegates: AppLocalizations.localizationsDelegates,
           supportedLocales: AppLocalizations.supportedLocales,
           theme: ThemeData(
+            colorScheme: Theme.of(context).colorScheme.copyWith(
+              secondary: theme.grey3,
+            ),
+            textSelectionTheme: Theme.of(context).textSelectionTheme.copyWith(
+                  selectionColor: theme.grey2,
+                  selectionHandleColor: theme.grey1,
+                ),
```

Comment:
If you want to change only few of many required attributes inside an object.
Instead if creating a completely new object and manually setting dozens of required parameters, just take an existing object and replace it with an slightly modified one.

These are the only options, since the attributes of ColorScheme are final and can't be changed after being set.
But this is a common pattern in Flutter. Each model that extends AbstractDatabaseObject works also this way.